__tests__/app.test.js

```diff
@@ -12,34 +13,119 @@ return seed(testData)
 
 
 afterAll(() => {
-    if (db.end) db.end();
+    if (db.end){ 
+       return db.end()};
   });
 
+describe('Testing for bad path', () => {
+    test('should return 404 error', () => {
+        return request(app)
+        .get('/non-path')
+        .expect(404)
+        .then(({body: {msg}})=>{
+expect(msg).toBe('path not found')
+        })
+    });
+    
+});
 
-
-  describe('Get/api/categories', () => {
+  describe('Get /api/categories', () => {
     test('Status 200 - should respond with an array of category objects', () => {
         return request(app)
         .get("/api/categories")
         .expect(200)
         .then(({body})=>{
```

Comment:
I'd recommend you follow the same pattern for inline destructuring (like in the test above) for consistency